<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOclBM_params</class>
 <widget class="QWidget" name="QOclBM_params">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_basic">
        <property name="text">
         <string>Basic Preset</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_sobel">
        <property name="text">
         <string>Pre Filter X Sobel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string># Disparity</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_disp">
       <property name="minimum">
        <number>-300</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>64</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_disp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>WinSize</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_win">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="pageStep">
        <number>11</number>
       </property>
       <property name="value">
        <number>9</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_winsize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_disp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_disp</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_win</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_winsize</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
